source/system-administrators/activities/reindexing-content-in-prod.rst

```diff
@@ -4,95 +4,220 @@
 Reindexing Content Without Disrupting Service in Production
 ===========================================================
 
-In some scenarios it's not possible/appropriate to delete a live index and wait for the index to rebuild in production.  Perhaps the index is
-driving dynamic features on the site that will break while the index is empty or being rebuilt.  In these scenarios you need a process for building
-the index off line and swapping it in.
+In some scenarios it's not possible/appropriate to delete a live index and wait for the index to rebuild in production. 
+Perhaps the index is driving dynamic features on the site that will break while the index is empty or being rebuilt.  
+In these scenarios you need a process for building the index off line and swapping it in.
+
+-------------------
+Using Elasticsearch
+-------------------
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 1: Create a new index
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to create a new index on Elasticsearch where you can index the content:
+
+#. Check the next version for the index name, you can find the current name using the following command:
+   
+   .. code-block:: bash
+      :linenos:
+   
+      curl -s http://ES_HOST:ES_PORT/_cat/aliases | grep SITE_NAME
+     
+      # Example result
+     
+      SITE_NAME   SITE_NAME_v1   - - -
+
+#. Download the appropriate settings for the environment and Crafter CMS version being updated, for example the latest
```

Comment:
I think this line should say: _Download the Elasticsearch settings appropriate for the index. If the index is an authoring index (it has an `-authoring` suffix), then use the `authoring-index-settings.json`. If its any other index (it has a `-preview` suffix or no suffix), then use the `default-index-settings.json`_.